packages/core/src/modules/proofs/ProofsModule.ts

```diff
@@ -38,13 +34,5 @@ export class ProofsModule implements Module {
 
     // Proof Formats
     dependencyManager.registerSingleton(IndyProofFormatService)
```

Comment:
Here is the method where ProofsModule should register the features it supports. In this case you will have present-proof V1 and V2, so it should have something like:

```suggestion
// Features
    featureRegistry.register(
      new Protocol({
        id: 'https://didcomm.org/present-proof/1.0',
        roles: ['verifier', 'prover'],
      })
    )
    featureRegistry.register(
      new Protocol({
        id: 'https://didcomm.org/present-proof/2.0',
        roles: ['verifier', 'prover'],
      })
    )
```